### DISCUSSION TITLE
How to prevent spam account?

### DISCUSSION THREAD

**Discussion by Sahin-Albayram** [bounswe/bounswe2023group2#76] [Ideas]
When someone get banned ,from spamming, he/she can easily open guess account and continue to spam. How should we prevent this case ?

**Comment by higurbuzz** [comment]
I don't know if it can be done but IP address or device id can be banned.

**Comment by mehmetkuzu** [comment]
If E-mail is mandatory to register (and e-mail verification step for the registration is mandatory) no need to bother.
